pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "redshells"
+version = "0.0.0"  # using poetry-dynamic-versioning
+description='Tasks which are defined using gokart.TaskOnKart. The tasks can be used with data pipeline library "luigi".'
+authors = ["M3, inc."]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/m3dev/redshells"
+repository = "https://github.com/m3dev/redshells"
+documentation = ""
+
+[tool.poetry-dynamic-versioning]
+enable = true
+style = "pep440"
+pattern = "^(?P<base>\\d+\\.\\d+\\.\\d+)"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+luigi = "*"
```

Comment:
Maybe you don't need to write it, it depends on gokart.

Reply:
You're right! fixed!